heartbeat/LVM

```diff
@@ -320,6 +320,18 @@ LVM_status() {
 			fi
 		fi
 	fi
+
+	# Check if VG is still available (e.g. for multipath where the device
+	# doesn't disappear)
+	if [ "$LVM_MAJOR" -eq "1" ]; then
+		output=$(vgscan $vg 2>&1)
+	else
+		output=$(vgscan --cache 2>&1)
+	fi
```

Comment:
I think 'vgscan --cache' is very expensive lvm command, especially if there are a large of LVs on system. LVM_status will be invoked periodically. I don't think it's good idea to ad it in status checking.